examples/pcrglobwb.ipynb

```diff
@@ -2,28 +2,28 @@
  "cells": [
```

Comment:
<p>Can we show the use of setup, by giving some arguments like</p><p> </p><pre class="ql-syntax" spellcheck="false">cfg_file, cfg_dir = pcrglob.setup(landSurfaceOptions={'debugWaterBalance': 'False'})
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done, but I drastically reduced the configurable parameters (had discussed this with Niels a while ago), and made them use the 'ewatercycle' format instead of just exposing the full model config file.